browser/extensions/api/brave_wallet_api.cc

```diff
@@ -21,19 +20,27 @@
 #include "chrome/browser/profiles/profile.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/web_contents.h"
-#include "crypto/aead.h"
-#include "crypto/hkdf.h"
-#include "crypto/random.h"
-#include "crypto/symmetric_key.h"
 #include "extensions/browser/extension_registry.h"
 #include "extensions/browser/extension_util.h"
+#include "base/files/file_util.h"
```

Comment:
I think this is not used in this file, probably was when you were developing at one point. I think you can remove it.

Reply:
Ah yes, will remove

Reply:
removed